test/test_serialization.py

```diff
@@ -514,24 +514,33 @@ def load_bytes():
         gpu_last_map_locations = [
             f'cuda:{torch.cuda.device_count() - 1}',
         ]
+        xpu_0_map_locations = [
```

Comment:
Maybe we can unify `gpu_0_map_locations` and `xpu_0_map_locations` to a `def generate_map_location(device_type):` to generate the list.

Reply:
done